handler/src/test/java/io/netty/handler/ssl/SSLEngineTest.java

```diff
@@ -2163,7 +2180,10 @@ private static void testCloseInboundAfterBeginHandshake(SSLEngine engine) throws
         engine.beginHandshake();
         try {
             engine.closeInbound();
-            fail();
+            // Workaround for conscrypt bug
```

Comment:
Which bug is this? Is there an github issue or similar that could be linked to?

Reply:
@ejona86 I added links to the bugs as comments.